scripts/db.dropcolumn/db.dropcolumn.py

```diff
@@ -80,30 +80,44 @@ def main():
         return 0
 
     if driver == "sqlite":
-        # echo "Using special trick for SQLite"
-        # http://www.sqlite.org/faq.html#q13
-        colnames = []
-        coltypes = []
-        for f in gscript.db_describe(table)["cols"]:
-            if f[0] != column:
-                colnames.append(f[0])
-                coltypes.append("%s %s" % (f[0], f[1]))
-
-        colnames = ", ".join(colnames)
-        coltypes = ", ".join(coltypes)
-
-        cmds = [
-            "BEGIN TRANSACTION",
-            "CREATE TEMPORARY TABLE ${table}_backup(${coldef})",
-            "INSERT INTO ${table}_backup SELECT ${colnames} FROM ${table}",
-            "DROP TABLE ${table}",
-            "CREATE TABLE ${table}(${coldef})",
-            "INSERT INTO ${table} SELECT ${colnames} FROM ${table}_backup",
-            "DROP TABLE ${table}_backup",
-            "COMMIT",
-        ]
-        tmpl = string.Template(";\n".join(cmds))
-        sql = tmpl.substitute(table=table, coldef=coltypes, colnames=colnames)
+        sqlite3_version = gscript.read_command(
+            "db.select",
+            sql="SELECT sqlite_version();",
+            flags="c",
+            database=database,
+            driver=driver,
+        )
+
+        if sqlite3_version >= "3.35.0":
```

Comment:
Testing versions is a pain:

```python
>>> "10.0.0" >= "3.35.0"
False
>>> [int(i) for i in "10.0.0".split(".")] >= [int(i) for i in "3.35.0".split(".")]
True
```

but:

```python
>>> [int(i) for i in "4.0.0-dev".split(".")] >= [int(i) for i in "3.35.0".split(".")]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 1, in <listcomp>
ValueError: invalid literal for int() with base 10: '0-dev'
```

Reply:
Testing only major and minor version (3.35) should be enough and skip the patch/minor part which is not really relevant to this.

Reply:
Now comparing major and minor versions as arrays of INTs in recent commit.